test/hotspot/jtreg/vmTestbase/nsk/share/jdi/Binder.java

```diff
@@ -755,11 +754,9 @@ private Map<String,? extends Argument> setupLaunchingConnector(LaunchingConnecto
             vmArgs += " -Djdk.virtualThreadScheduler.parallelism=15";
         }
 
-/*
-        if (classPath != null) {
+        if (classPath != null && !vmArgs.contains("-cp") && !vmArgs.contains("-classpath")) {
```

Comment:
How does this change relate to using driver instead of othervm?

Reply:
The classpaths are different for tests executed in agentvm and othervm.
As I see from logs jtreg uses CLASSPATH in othervm mode so debugee also inherits it. While agentvm load classed from 'test.class.path'.
It is not harm to use classpath 'test.class.path' in othervm mode.